_sass/event/_elements.scss

```diff
@@ -320,12 +323,12 @@
     a i {
       color: white;
       opacity: 0.5;
-      -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+      -ms-filter: 'progid:DXImageTransform.Microsoft.Alpha(Opacity=50)';
```

Comment:
This was [removed in IE10](https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-filter), and all MS browsers apparently support `opacity` so we should be safe to remove the usages of `-ms-filter`?

Reply:
(can't believe that needed to be used in the past 😨)